Source/Tanks/ModuleFuelTanksRF.cs

```diff
@@ -472,6 +472,24 @@ private IEnumerator CalculateTankBoiloff(double deltaTime, bool analyticalMode =
             highlyPressurized = def.highlyPressurized;
             numberOfMLILayers = def.numberOfMLILayers;
 
+            if (def.maxMLILayers >= 0f)
```

Comment:
Ok so you're overriding the module's defined maximum with the tank def's value if it's 0 or greater and default is -1 but what happens then if another tank type is selected which has the default value? maxMLILayers should revert back to the module's defined value.

Also, just an FYI but persisting anything in TANK_DEFINITION doesn't really do anything. I'm not sure why that ever happened. I guess the original was written that way and everyone else just follows suit (me included) It's probably harmless.

Now, if you were to do the same thing in ModuleFuelTanks with the three values that you persisted, you would probably regret it because if you ever decided decided that a given tank's default values should change then those changes would not propagate to player's saved craft and saved game files. (or to RO/RP saved craft for that matter)

Reply:
Yes, you're probably right that it would be better to reset the values back to Tank's default if TANK_DEFINITION doesn't do anything. On the side of RO Pap already explicitly defined the values for all tank definitions so it wouldn't be an issue there.
Aren't the fields on TANK_DEFINITION made Persistent just so that these could easily be deserialized from ConfigNodes?

Reply:
It might not be an issue for RO but I have to think more broadly because this code can potentially affect more than just RO.

Persistent... I must have been thinking about KSPField, disregard.

Reply:
Added a commit that reverts the values back to what the ModuleFuelTanks originally had before being overwritten by TANK_DEFINITION